layouts/partials/redirect.html

```diff
@@ -0,0 +1,12 @@
+{{ if .Params.redirect_to }}
+  {{ $redirect_to := .Params.redirect_to }}
+  {{ $redirect_after := .Params.redirect_after | default "0" }}
+  <meta http-equiv="refresh" content="{{ $redirect_after }}; url={{ $redirect_to }}">
+  <link rel="canonical" href="{{ $redirect_to }}">
+  <script>
+    var redirectTimeout = parseInt("{{ $redirect_after }}") * 1000;
```

Comment:
Since this is a number, wouldn't the substitution look like this (for, say, 3):

```js
var redirectTimeout = parseInt("3") * 1000;
```

Wouldn't it be better to do something like:

```js
var redirectTimeout = {{ $redirect_after }} * 1000;
```

Which would work out to `var redirectTimeout = 3 * 1000;` (reusing my 3 second timeout example)

Reply:
I think your suggestion is more ideal if it works! I am unfamiliar with this templating language and using the variable without quotes showed an error in my IDE, will give this a try!

Reply:
It works! But VSCode is angy even after I installed an [extension](https://marketplace.visualstudio.com/items?itemName=budparr.language-hugo-vscode) to support HUGO syntax highlighting. Any suggestions? 
<img width="1115" alt="Screen Shot 2022-09-21 at 6 47 20 PM" src="https://user-images.githubusercontent.com/2709526/191624430-549213ee-4a29-49ff-b2f6-a4bd54a3049a.png">

Reply:
I've not been happy with the highlighting situation in hugo in tools like vscode because of edge cases like this, and I've hit my own edge cases where the highlighting goes crazy; but the code is *technically* correct since it's hugo -> html, and the final output matters the most in this case.

Also, when you started using vscode with the repo, did it recommend some extensions? Because that extension is one of the [recommended](https://github.com/The-Balance-FFXIV/glam/blob/e70e11d8a82f46115d0fcb99ace79e4691daec2b/.vscode/extensions.json#L3) ones anyway.